nservicebus/concept-overview.md

```diff
@@ -11,53 +11,67 @@ A high level overview of the concepts, features and vernacular of NServiceBus.
 
-### [Header](/nservicebus/messaging/headers.md)
+### Endpoint
+
+An Endpoint is a design-time concept that has a name and a collection of associated Message Handlers and Sagas. A definition of an Endpoint lives as a group of artefacts in a source code repository. An Endpoint (the binaries that result from building the endpoint's source code) can be deployed to a number of machines and environments. Each such deployment is a separate Endpoint Instance.
+
+### Endpoint Instance
+
+An Endpoint Instance is a run-time object that allows to interact with the bus. Endpoint Instances are able to send and receive messages (an exception is a *send-only* endpoint). It runs associated Message Handlers and Sagas to process incoming messages. An Endpoint Instance has a single Input Queue (which can be shared with other instances of the same Endpoint in a scale-out scenario). 
+
+### [Hosting](/nservicebus/hosting)
+
+The act of Hosting refers to running an Endpoint Instance in some process. This can be any .NET process such as a console application, a website, or a Windows service. Multiple Endpoint instances can be hosted in a single process (custom AppDomains are required to achieve this in version 4 and below).
 
-Extra information about a message is communicated over the transport in a serialized collection of key value pairs in a similar way to how http headers are used.
+### [Transport](/nservicebus/transports/)
+
+The Transport is the mechanism that NServiceBus Endpoint Instances use to communicate with each other. NServiceBus supports many different transports in a pluggable manner. For Endpoint Instances to communicate they need to share a common Transport technology.
 
+### Input Queue
+
+Each endpoint instance is assigned a single Input Queue. This queue can be shared among instances of same Endpoint but never with other Endpoints.
 
 ### [Publish Subscribe](/nservicebus/messaging/publish-subscribe)
 
 Publish Subscribe is the interaction of 
 
- * Registering interest in being notified about an Event.
- * That Event being delivered to the endpoint that registered
-
-A "Subscriber" lets the "Publisher" know they're interested in an Event, and the "Publisher" stores the "Subscriber" address so that it knows where to send the Event. 
+ * Registering interest in being notified about an event (*subscriber*).
+ * That event being delivered to the endpoint that registered itself (*publisher*)
 
+Both *publisher* and *subscriber* are Endpoints which means they are logical rather than physical (e.g. `Sales` or `Shipping`, not `Cache-1` or `Web-1`). Each event is delivered to a single Instance of each *subscriber* Endpoint.
```

Comment:
how about "Each event is delivered to a **single Endpoint Instance** of each subscribed Endpoint"?